src/ape/managers/chain.py

```diff
@@ -1092,6 +1093,9 @@ def instance_at(
             :class:`~ape.contracts.base.ContractInstance`
         """
 
+        if self.network_manager.active_provider is None:
```

Comment:
this happens automatically when doing `self.provider`, so it probably isnt needed here